index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+   <head>
+     <title>Dr. Borlaug's life</title>
+     <link rel="stylesheet" href="style.css">
+   </head>
+   <body>
+     <main id="main">
+       <h1 id="title">Dr. Norman Borlaug</h1>
```

Comment:
Kindly avoid using HTML sematic tags as ID names

Reply:
I am sorry to say that but tags (`<div>`, `</i>`, etc.) cannot be used as id names but tag names can. Kindly explain why the requested change is made. Kindly give an important and easy-to-understand reason. 

To be honest I don't see a good reason for this change. But maybe you'll find one. 

If I were you I would try to find another reason or another change request instead of this one.